catalog/transformer/catalog-transformer-streaming-impl/src/main/java/org/codice/ddf/transformer/xml/streaming/impl/GmlHandler.java

```diff
@@ -82,7 +97,12 @@ public GmlHandler(GMLHandler gmlHandler, Gml3ToWkt gml3Converter) {
      */
     @Override
     public List<Attribute> getAttributes() {
-        return attributes;
+        return saxEventHandlerUtils.getCombinedMultiValuedAttributes(getSupportedAttributeDescriptors(), attributes);
+    }
+
+    @Override
+    public Set<AttributeDescriptor> getSupportedAttributeDescriptors() {
```

Comment:
Moving getSupportedAttributeDescriptors to the abstract class. Leaving getAttributes alone for now because the default implementation would use the SaxEventHandlerUtils class which is in streaming-lib. The abstract class lives in the api. Don't want to tie the api to streaming-lib. The abstract class should find a new home outside the api.